### DISCUSSION TITLE
Custom achievements for organizations

### DISCUSSION THREAD

**Discussion by alx-andru** [community/community#19861] [Other Feature Feedback, Questions, & Ideas]
Would love to create achievements that are specific to organizations e.g. to celebrate the first PR merge of a new employee

**Comment by d-coder111** [comment]
yeah even i think that would be really cool😍

**Reply by vunguyenhuychuong** [reply]
really awsome to show off them in profile 👍

**Reply by simonmazzaroth** [reply]
I saw your feature request and I couldn’t help but share your enthusiasm! Just like you, we’re big fans of GitHub’s achievements and its API - they truly set a high standard for the developer community.

I would like to gently mention that I am part of the Vaunt team and this platform is closely related to what you are looking for. Vaunt is designed to help you create custom achievements to enhance open-source project participation and community building, leveraging GitHub’s outstanding API to provide more insights and tools for project management and community interaction.

While we are independent of GitHub, we strive to complement the great work they do. If you are interested in exploring this further, please feel free to check us out or contact us for more information. We’re always excited to connect with other open-source enthusiasts and learn how we can add value to the community.

**Comment by JhonnyLusonode** [comment]
I would love to see that implemented honestly! 👍

**Comment by alx-andru** [comment]
We often celebrate very random things, in one repo we reached 1000 commits and gave a shoutout to everyone contributing. 
I could see it very rewarding for folks that do great code reviews, help build out our documentation etc. Those achievements could be only visible to the organization.

**Comment by jlumbroso** [comment]
I think having a marketplace of GitHub Badges similarly to GitHub Actions and Apps would be very much in the spirit of GitHub.

Beyond having organizations develop internal rewards, I think this would make it attractive for universities (and high schools) to use GitHub for the same purpose.

**Comment by JamesManningR** [comment]
I'd love to see this. We've got some projects that require specific skills to engage with and I'd love to mark that for people on ther GH profile

**Comment by JesseAA** [comment]
This could be a feature of GitHub Enterprise (server/cloud) too

**Comment by dterol23** [comment]
What about linking this custom achievements with virtual badges issued by 3rd parties https://www.holopin.io/?

**Comment by grentank** [comment]
What a neat idea! This could be especially useful for IT education organizations. Gamification can build additional motivation and interest amongst students.

**Comment by micheal-w-wells** [comment]
would use this for hackathon event awards

**Comment by Yanjia0** [comment]
would be great if we could create our achievements and give them to our community members~

**Comment by AleksandrDruk** [comment]
cool

**Comment by KarmaPoliceT2** [comment]
I'm interested in this too... as a part of an open source organization I'd love for us to be able to give out achievements to significant contributors or specific achievements within the project... +1
